examples/notebooks/introduction_enlyze_python_sdk.ipynb

```diff
@@ -0,0 +1,2981 @@
+       "2 2022-05-17 04:00:20+00:00                     388.0                  127.0   \n",
+       "3 2022-05-17 04:00:30+00:00                     388.0                  127.0   \n",
+       "4 2022-05-17 04:00:40+00:00                     388.0                  127.0   \n",
+       "\n",
+       "   Extruder A Durchsatz  Extruder A Folienstärke  \\\n",
+       "0                  50.0                      2.0   \n",
+       "1                  50.0                      2.0   \n",
+       "2                  50.0                      2.0   \n",
+       "3                  50.0                      2.0   \n",
+       "4                  50.0                      2.0   \n",
+       "\n",
+       "   Extruder A Schmelzetemperatur  Extruder B Folienstärke  \\\n",
+       "0                          211.0                      3.0   \n",
+       "1                          211.0                      3.0   \n",
+       "2                          211.0                      3.0   \n",
+       "3                          211.0                      3.0   \n",
+       "4                          211.0                      3.0   \n",
+       "\n",
+       "   Extruder B Schneckenumdrehung  Extruder B Schmelzedruck  \\\n",
+       "0                           71.0                     502.0   \n",
+       "1                           71.0                     502.0   \n",
+       "2                           71.0                     502.0   \n",
+       "3                           71.0                     502.0   \n",
+       "4                           71.0                     500.0   \n",
+       "\n",
+       "   Extruder A Schneckenumdrehung  Extruder B Durchsatz  Extruder A Förderrate  \\\n",
+       "0                           67.0                  90.0                   74.0   \n",
+       "1                           67.0                  90.0                   74.0   \n",
+       "2                           67.0                  90.0                   74.0   \n",
+       "3                           67.0                  90.0                   74.0   \n",
+       "4                           67.0                  90.0                   74.0   \n",
+       "\n",
+       "   Extruder B Schmelzetemperatur  \n",
+       "0                          210.0  \n",
+       "1                          210.0  \n",
+       "2                          210.0  \n",
+       "3                          210.0  \n",
+       "4                          210.0  "
+      ]
+     },
+     "execution_count": 29,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "data = data.reset_index()\n",
+    "data.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e2ac6c18-da31-4ebc-869c-b4c2b0911d3c",
+   "metadata": {},
+   "source": [
+    "To convert the timezone back to the local timezone you can use pandas Datetime functionality:"
```

Comment:
Is that a regular usage pattern? We're returning UTC because, AFAIK, pandas (or rather numpy underneath) doesn't like inhomogenous timezone offset, which regularily happens when daylight saving time is switched.

Reply:
It depends on the use case. I'm not so sure about the inhomogeneous timezone offsets, but so far, I had no problem with either. If I only want to plot it, I will directly convert it to the local time zone. If I do additional mapping with other data, I will do that in utc and later convert it to the local time zone. My standard procedure is actually to immediately reset the index and create a second column, `time_local`, with the time zone of the machine. Often I even need to remove the time zone info for different visualizations or exports.

Reply:
> My standard procedure is actually to immediately reset the index and create a second column, time_local, with the time zone of the machine.

Converting to naive local time is indeed something that I see you'd do all the time when interactively working with the data. I'll create a ticket to add that to the `to_dataframe()` method as a convenience option.

Reply:
#33